apps/frontend-manage/src/components/questions/tags/SuspendedTags.tsx

```diff
@@ -46,55 +72,70 @@ function SuspendedTags({ showUntagged, activeTags, handleTagClick }: Props) {
     )
 
   return (
-    <ul className="flex min-h-[4.7rem] list-none flex-col overflow-y-auto">
-      {data.userTags.map(
-        (tag: Tag, ix): React.ReactElement => (
-          <UserTag
-            key={tag.id}
-            tag={tag}
-            handleTagClick={(tag: string) =>
-              handleTagClick({
-                tagName: tag,
-                isTypeTag: false,
-                isStatusTag: false,
-                isUntagged: false,
-              })
-            }
-            active={activeTags.includes(tag.name)}
-            onMoveDown={
-              ix < data.userTags!.length - 1
-                ? () =>
-                    updateTagOrdering({
-                      variables: { originIx: ix, targetIx: ix + 1 },
-                    })
-                : undefined
-            }
-            onMoveUp={
-              ix > 0
-                ? () =>
-                    updateTagOrdering({
-                      variables: { originIx: ix, targetIx: ix - 1 },
-                    })
-                : undefined
-            }
-          />
-        )
-      )}
-      <UserTag
-        key={'untagged-tag-trigger'}
-        tag={{ id: 0, name: t('manage.questionPool.untagged'), order: 1 }}
-        handleTagClick={(tag: string) =>
-          handleTagClick({
-            tagName: tag,
-            isTypeTag: false,
-            isStatusTag: false,
-            isUntagged: true,
-          })
-        }
-        active={showUntagged}
-        isStatic
+    <div>
+      <TextField
+        placeholder={t('manage.general.searchPlaceholder')}
+        value={searchInput}
+        onChange={(newValue: string) => {
+          setSearchInput(newValue)
+          handleSearch(newValue)
+        }}
+        icon={faMagnifyingGlass}
+        className={{
+          input: 'h-10 pl-9',
+          field: 'w-30 rounded-md pr-3',
+        }}
       />
-    </ul>
+      <ul className="flex min-h-[4.7rem] list-none flex-col overflow-y-auto">
+        {filteredTags.map(
+          (tag: Tag, ix): React.ReactElement => (
+            <UserTag
+              key={tag.id}
+              tag={tag}
+              handleTagClick={(tag: string) =>
+                handleTagClick({
+                  tagName: tag,
+                  isTypeTag: false,
+                  isStatusTag: false,
+                  isUntagged: false,
+                })
+              }
+              active={activeTags.includes(tag.name)}
+              onMoveDown={
```

Comment:
Maybe we should disable the sorting functionality here whenever the search is active, since sorting does not have any visible effect in these cases?

Reply:
not quite sure what you mean. for me the manual "move up" and "move down" buttons work when search is active